float64 a
float64 k
string market
string objective
uint32 lighthouseFee
uint32 price_range
---
string result
